src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java

```diff
@@ -82,11 +82,10 @@ public void loggedInUserOnMobileCanVoteForFirstReply() {
     firstReplyUpvoteButtonClickAddsAnUpvoteAndSecondClickRemovesTheUpvote();
   }
 
-  @Test(groups = "discussions-loggedInUserOnMobileCanUpvote", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-loggedInUserOnMobileCanUpvote")
   @Execute(asUser = User.USER_3)
   @InBrowser(browserSize = MOBILE_RESOLUTION)
-  public void loggedInUserOnMobileCanUpvote() {
+  public void regularUserOnMobileCanUpvote() {
```

Comment:
I'd like us to move our custom vocabulary (how we call users) to wikia user access level vocabulary:
http://mediawiki119.wikia.com/wiki/Help:User_access_levels
This vocabulary is already used in our Test User pages
https://wikia-inc.atlassian.net/wiki/display/SOC/Manual+testing

The goal is to be more precise. Logged in user doesn't tell much about who the user is. Is this staff ? Is this a helper ? In this case this is the regular user (no additional rights granted)